types/frisby/index.d.ts

```diff
@@ -48,17 +48,29 @@ export class FrisbySpec {
     inspectStatus(): FrisbySpec;
     patch(url: string, params?: {}): FrisbySpec;
     post(url: string, params?: {}): FrisbySpec;
-    promise(): Promise<nodeFetch.Response>;
+    promise(): Promise<FrisbyResponse>;
```

Comment:
Actually,  `promise()` function returns `FrisbyResponse`, not `nodeFetch.Response`. So I fixed.

https://github.com/vlucas/frisby/blob/master/src/frisby/spec.js#L303